webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator_unit_test.go

```diff
@@ -1019,16 +1105,16 @@ func unitTestsValidateCreate() {
 						ctx.vm.Spec.Network.Interfaces[0].SearchDomains = []string{"dev.local"}
 					},
 					validate: doValidateWithMsg(
-						`spec.network.interfaces[0].searchDomains: Invalid value: "dev.local": searchDomains is available only with the following bootstrap providers: CloudInit LinuxPrep and Sysprep (except for RawSysprep)`,
+						`spec.network.interfaces[0].searchDomains: Invalid value: "dev.local": searchDomains is available only with the following bootstrap providers: CloudInit`,
 					),
 				},
 			),
 
-			Entry("validate searchDomains when bootstrap supports searchDomains",
+			Entry("allows searchDomains with CloudInit",
```

Comment:
Do you mean `Allows per-device searchDomains with CloudInit`? Do you mind changing the entry description to make this more clear?